static/app/components/replays/replayController.tsx

```diff
@@ -170,10 +186,21 @@ const ReplayControls = ({
   toggleFullscreen = () => {},
   speedOptions = [0.1, 0.25, 0.5, 1, 2, 4],
 }: Props) => {
+  const config = useLegacyStore(ConfigStore);
+  const organization = useOrganization();
   const barRef = useRef<HTMLDivElement>(null);
   const [compactLevel, setCompactLevel] = useState(0);
   const {isFullscreen} = useFullscreen();
 
+  const handleFullscreenToggle = () => {
+    trackAdvancedAnalyticsEvent('replay.toggle-fullscreen', {
+      organization,
+      user_email: config.user.email,
+      fullscreen: !isFullscreen,
+    });
```

Comment:
i would also shove this inside the `toggleFullscreen` prop

because like, if someone makes a call like this we shouldn't be logging: `<ReplayControls toggleFullscreen={() => void} />`